### PR TITLE
Animartrix updates

### PR DESCRIPTION
**Pull request by netmindz**
- Usermod to be disabled by default to not clutter the list of effects too much
- Disabling actually preventing the effects being listed (reboot required)
- Updated library with fix for SM1
- Pass speed slider to animartrix speed factor

### CODE REVIEW

File: usermods/usermod_v2_animartrix/usermod_v2_animartrix.h [hunk 1]
```diff
@@ -67,6 +67,8 @@ class ANIMartRIXMod:public ANIMartRIX {
 	  if (SEGENV.call == 0) {
 		init(SEGMENT.virtualWidth(), SEGMENT.virtualHeight(), false);
 	  }
+	  float speedFactor = map(SEGMENT.speed, 0, 255, 1, 100) / 10;
```
Comment (netmindz) [inline usermods/usermod_v2_animartrix/usermod_v2_animartrix.h]: This calc isn't right. 

Requirements:
0 = 0.1
125 = 1.0
255 = 10.0

Problem: 125 currently returns 5.0

Comment (softhack007) [reply usermods/usermod_v2_animartrix/usermod_v2_animartrix.h]: I think the easiest solution would be to split the mapping into two parts: <1.0 and >=1.0.
Also you need to use "mapf" as the mapped result is float.
 something like 
``` C++
float speedFactor = 1.0f:
if (SEGMENT.speed < 128) speedFactor = mapf(SEGMENT.speed,   0, 127, 0.1f, 1.0f);
                    else speedFactor = mapf(SEGMENT.speed, 128, 255, 1.0f, 10.0f);
```

The small overlap at "1.0f" does not matter much - nobody will notice there is no difference between 127 and 128.

Comment (netmindz) [reply usermods/usermod_v2_animartrix/usermod_v2_animartrix.h]: Ah, didn't know about mapf (). Thought I might need to split into increase Vs decrease

Comment (softhack007) [reply usermods/usermod_v2_animartrix/usermod_v2_animartrix.h]: you can grab `mapf()` here

https://github.com/MoonModules/WLED/blob/17147f3b5014fedf82a5ed6bd2aa75fedf2109e0/wled00/FX.cpp#L6205-L6209

actually its planned to make this function global, as WLED is starting to collect several copies of mapf() in unsermods..

**Pull request merged**